### ISSUE TITLE
Suggestion: lighter color to indicate string parse error

### ISSUE THREAD

**Issue by BioTurboNick** [fonsp/Pluto.jl#749]
I suggest making the error color for string parsing (i.e. when missing end quote mark) lighter. It's hard to read the text currently.

![image](https://user-images.githubusercontent.com/1438610/101292249-3ff0a380-37dc-11eb-8941-17578f584946.png)

**Comment by pankgeorg** [comment]
That's a good suggestion. Current contrast ratio is 1.5, much lower than the [recommended](https://www.w3.org/WAI/WCAG21/Techniques/general/G145). I'll sketch up something soon!
